src/control/server/storage/scm/ipmctl.go

```diff
@@ -145,13 +161,39 @@ func (r *cmdRunner) checkIpmctl(badList []semVer) error {
 	return nil
 }
 
+func (cr *cmdRunner) createNamespace() (string, error) {
+	return cr.runCmd(cmdCreateNamespace)
```

Comment:
Are we eventually planning to shift these over to use the ndctl API instead of shelling out?

Reply:
I think we should for both ipmctl and ndctl, the only tricky bit will be switching to use the set goal ipmctl API as that looked nontrivial last time I looked. Think the task has just been bumped off the bottom of the ZZB list. Maybe we should reinstate the ticket @mjmac ?

Reply:
Create goal is tricky and might be worth continuing to push off. Everything else should be pretty trivial though.